tests/ffi/dev-mem_not_present/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+. ../../e2e/lib/utils
+
+check_dev-mem_not_present(){
```

Comment:
Is it a convention to use dash `-` in the function name? 
I searched online, seems it's allowed in bash, but not recommended.

Reply:
`-` is part of the "portable character set" [1] and thus allowed by at least POSIX [2]

[1] https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap06.html#tag_06_01
[2] https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_235

Reply:
Thank you both for the careful review.
I was trying to distinguish /dev/mem from other words, but it does look a bit weird, so I changed hyphen to low-line.

Reply:
```
3.235 Name
In the shell command language, a word consisting **solely of underscores, digits, and alphabetics** from the portable character set. The first character of a name is not a digit.
```

Reply:
> ```
> 3.235 Name
> In the shell command language, a word consisting 
> **solely of underscores, digits, and alphabetics** 
> from the portable character set. 
> The first character of a name is not a digit.
> ```

I did not read carefully enough. Thanks @qiyuann